### PR TITLE
tree-wide: Switch tabs ⭾ in various files over to spaces ␠

### PR DESCRIPTION
**Pull request by cgwalters**
As $DEITY intended.

I was reading the `prepare-root.c` code and the indentation damage was
distracting. Squash tabs that have leaked into various places in the code. I
didn't yet touch the `src/libostree` bits as that has higher potential for
conflict.

### CODE REVIEW

File: src/switchroot/ostree-prepare-root.c [hunk 1]
```diff
@@ -239,10 +239,10 @@ main(int argc, char *argv[])
        * later boot and `systemd-remount-fs.service`.
        */
       if (path_is_on_readonly_fs ("."))
-	{
-	  if (mount (".", ".", NULL, MS_REMOUNT | MS_SILENT, NULL) < 0)
-	    err (EXIT_FAILURE, "failed to remount rootfs writable (for overlayfs)");
-	}
+  {
```
Comment (jlebon) [inline src/switchroot/ostree-prepare-root.c]: This looks off.

File: src/rofiles-fuse/main.c [hunk 2]
```diff
@@ -513,39 +513,39 @@ static void
...
   fprintf (stdout,
-	   "usage: %s basepath mountpoint [options]\n"
-	   "\n"
-	   "   Makes basepath visible at mountpoint such that files are read-only, directories are writable\n"
-	   "\n"
-	   "general options:\n"
-	   "   -o opt,[opt...]     mount options\n"
-	   "   -h  --help          print help\n"
-	   "\n", progname);
+     "usage: %s basepath mountpoint [options]\n"
```
Comment (jlebon) [inline src/rofiles-fuse/main.c]: These look off too.

File: src/rofiles-fuse/main.c [hunk 3]
```diff
@@ -366,7 +366,7 @@ callback_read_buf (const char *path, struct fuse_bufvec **bufp,
 
 static int
 callback_read (const char *path, char *buf, size_t size, off_t offset,
-	       struct fuse_file_info *finfo)
+         struct fuse_file_info *finfo)
```
Comment (jlebon) [inline src/rofiles-fuse/main.c]: Did you mean these to be less indented? Blasphemy!

**Comment by jlebon** [comment]
I used to be in the "tabs for indentation, spaces for alignment" camp, though it's hard to make editors smart about it, so these days I'm all spaces as well.

**Comment by cgwalters** [comment]
https://www.youtube.com/watch?v=SsoOG6ZeyUI

**Comment by cgwalters** [comment]
Ok, fixup :arrow_up:

File: src/rofiles-fuse/main.c [hunk 4]
```diff
@@ -438,14 +438,14 @@ callback_access (const char *path, int mode)
...
-		   size_t size, int flags)
+                   size_t size, int flags)
 {
   return -ENOTSUP;
 }
 
 static int
 callback_getxattr (const char *path, const char *name, char *value,
-		   size_t size)
+       size_t size)
```
Comment (jlebon) [inline src/rofiles-fuse/main.c]: This one seems to have escaped the fixup roundup.

Comment (cgwalters) [inline src/rofiles-fuse/main.c]: Done, I also ran it through the Emacs `indent-region` while here, since it was almost clean.

**Comment by jlebon** [comment]
@rh-atomic-bot r+ da07990

**Comment by cgwalters** [comment]
Possibly a victim of restarting homu.

@rh-atomic-bot retry

**Comment by cgwalters** [comment]
@rh-atomic-bot retry

**Pull request closed without merging**